fala/templates/adviser/adviser_base.html

```diff
@@ -97,4 +97,13 @@ <h1 class="govuk-heading-xl govuk-!-margin-top-7">
       );
     });
   </script>
+
+  <script nonce="{{request.csp_nonce}}">
+    const printResultsPage = document.getElementById("printButton");
```

Comment:
Don't do this please, bad practice. Please create a `print.js` file and concat it with the govuk js file being imported into the project. 

Keeping JavaScript in separate files makes the code easier to read, maintain, and debug. It allows developers to work on the HTML structure and JavaScript logic independently, leading to cleaner and more organised code.

JavaScript code in a separate file can be reused across multiple pages of a website or even across different projects. This reduces code duplication and makes updates easier.

Finally debugging. I cannot debug inline JS.

Reply:
Will do. It's such a small bit of JS that I thought I could get away with it, but if i's bad practice will put into it's own file. 👍